core/src/main/scala/org/apache/gearpump/cluster/ClusterMessage.scala

```diff
@@ -155,26 +158,20 @@ object MasterToClient {
 
   /** Return the last error of this streaming application job */
   case class LastFailure(time: TimeStamp, error: String)
-}
 
-trait AppMasterRegisterData
+  sealed trait ApplicationResult
```

Comment:
how is `ApplicationResult` mapped to `ApplicationStatus` ? If both of them are internal interfaces, can we just return `ApplicationStatus` to the client ?